bin/skel.py

```diff
@@ -68,7 +69,12 @@ def _ask_variable_choice(name, choices, default=None):
     else:
         default = default or choices[0]
         text = f" ({default})" if default else ""
-        return _ask_variable(name + text, default if default else "")
+        while True:
+            got = _ask_variable(name + text, default if default else "")
+            if got in choices:
+                return got
+            else:
+                warn(f"unknown license: {got}")
```

Comment:
`_ask_variable_choice` is also used for things other than license, so this warning is not complete.